jerry-debugger/src/cli/cli.ts

```diff
@@ -45,17 +48,24 @@ export function getOptionsFromArgs(argv: Array<string>) {
   return {
     ...getHostAndPort(target),
     verbose: args['verbose'],
-    jsfile: args._[0],
+    address: args._[0] || '',
+    jsfile: args['client-source'] || 'untitled.js',
   };
 }
 
 export function main(proc: NodeJS.Process) {
   const options = getOptionsFromArgs(proc.argv.slice(2));
-  const msg = `Debugger listening on ws://${options.host}:${options.port}\n`;
-  proc.stdout.write(msg);
-  new ChromeDevToolsProxyServer({
-    host: options.host,
-    port: options.port,
-    jsfile: options.jsfile,
+  const jdebug = new JerryDebugger(options.address);
+  const debuggerUrl = `ws://${jdebug.host}:${jdebug.port}`;
+  jdebug.getConnectPromise().then(() => {
+    proc.stdout.write(`Connected to debugger at ${debuggerUrl}\n`);
+    proc.stdout.write(`Proxy listening at ws://${options.host}:${options.port}\n`);
+    new ChromeDevToolsProxyServer({
+      host: options.host,
+      port: options.port,
+      jsfile: options.jsfile,
+    });
+  }).catch(() => {
```

Comment:
Maybe grab the error and print it too?

`catch((e) => { ... // do something with e }`